src/demo-app/datepicker/datepicker-demo.html

```diff
@@ -38,6 +38,14 @@ <h2>Options</h2>
     <mat-datepicker #startAtPicker [touchUi]="touch" [disabled]="datepickerDisabled"></mat-datepicker>
   </mat-form-field>
 </p>
+<p>
+  <mat-form-field>
+    <input matInput [matDatepicker]="customCalendarHeaderPicker" placeholder="Custom calendar header"
```

Comment:
Instead of `placeholder="Custom calendar header"` can you do `<mat-label>Custom calendar header</mat-label>`

Reply:
done in 6d0a4d2264e085c011770bf3de0fe381598dbe33